.gitignore

```diff
@@ -33,4 +36,4 @@
 
 
 # emacs save files
```

Comment:
COOL! I thought you have been using vim.

Reply:
Aha, it's added by Dan, he uses emacs.